gammapy/modeling/models/spectral.py

```diff
@@ -213,6 +213,40 @@ def evaluate_error(self, energy, epsilon=1e-4):
         """
         return self._propagate_error(epsilon=epsilon, fct=self, energy=energy)
 
+    @property
+    def pivot_energy(self):
+        """The decorrelation energy for a given spectral model calculated numerically.
```

Comment:
It is a little weird to me that the function is named `pivot_energy` and that the description is `The decorrelation energy`. I know that the name `pivot_energy` was already in the code before. But maybe for clarity it is worse to rename everything coherently ?